transport/tchannel/channel_inbound.go

```diff
@@ -102,3 +103,13 @@ func (i *ChannelInbound) Stop() error {
 func (i *ChannelInbound) IsRunning() bool {
 	return i.once.IsRunning()
 }
+
+// Introspect returns the state of the inbound for introspection purposes.
+func (i *ChannelInbound) Introspect() introspection.InboundStatus {
+	c := i.transport.Channel()
+	return introspection.InboundStatus{
+		Transport: "tchannel",
+		Endpoint:  i.transport.ListenAddr(),
+		State:     c.State().String(),
```

Comment:
Not possible here, we might want a dedicated debug page for tchannel, here is is way too much information to shoehorn into a common introspection summary.